README.md

```diff
@@ -17,3 +17,35 @@ a few minutes and will help us better support the Blockly community.
 Cross-browser Testing Platform and Open Source <3 Provided by [Sauce Labs](https://saucelabs.com)
 
 Want to contribute? Great! First, read [our guidelines for contributors](https://developers.google.com/blockly/guides/modify/contributing).
+
+## Releases
+
+We release by pushing the latest code to the master branch, followed by updating our [docs](developers.google.com/blockly) and [demo pages](https://blockly-demo.appspot.com). We typically release a new version of Blockly once a quarter (every 3 months). If there are breaking bugs, such as a crash when performing a standard action or a rendering issue that makes Blockly unusable, we will cherry-pick fixes to master between releases to fix them.
+
+Releases are tagged by the release date (YYYYMMDD) with a leading '1.' and a trailing '.0' in case we ever need a major or minor version. For example, [1.20190419.0](https://github.com/google/blockly/tree/1.20190419.0)
+
+### New APIs
+
+Once a new API is merged into master it is considered beta until the following release. We generally try to avoid changing an API after it has been merged to master, but sometimes we need to make changes after seeing how an API is used. If an API has been around for at least two releases we'll do our best to avoid breaking it.
+
+### Branches
+
+There are two main branches for Blockly.
+
+**[master](https://github.com/google/blockly)** - This is the (mostly) stable current release of Blockly.
+
+**[develop](https://github.com/google/blockly/tree/develop)** - This is where most of our work happens. Pull requests should always be made against develop. This branch will generally be usable, but may be less stable than the master branch. Once something is in develop we expect it to merge to master in the next release.
+
+**other branches:** - Larger changes may have their own branches until they are good enough for people to try out. These will be developed separately until we think they are almost ready for release. These branches typically get merged into develop immediately after a release to allow extra time for testing.
+
+## Issues and Milestones
+
+We typically triage all bugs within 2 working days, which includes adding any appropriate labels and assigning it to a milestone.
+
+### Milestones
+
+**Upcoming release** - The upcoming release milestone is for all bugs we plan on fixing before the next release. This typically has the form of year_quarter_release. For example, 2019_q2_release. Some bugs will be added to this release when they are triaged, others may be added closer to a release.
```

Comment:
Add backticks around `year_quarter_release` and `2019_q2_release`

Reply:
done.